### PR TITLE
Add contribution guidelines

### PR DESCRIPTION
**Pull request by neilbmclaughlin**
Raising a desirable change with no associated issue.

### CODE REVIEW

**Comment by neilbmclaughlin** [comment]
Also, forced my to sync my fork with the original repo following the PR merges.

**Comment by arctangent** [comment]
In my role as Approver, I approve this change as a good fit

**Comment by arctangent** [comment]
@damonjagger please review this code (and reject it because it contains malicious code)

**Comment by arctangent** [comment]
And also create an issue to match the PR

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!--
+Thanks for wanting to report an issue.
+
+In order for the issue to be resolved as quickly as possible please provide as detailed information as possible.
+
+Use the following headings as a guide.
+-->
+## What is the issue?
```
Comment (damonjagger) [inline .github/ISSUE_TEMPLATE.md]: Malicious code

**Comment by arctangent** [comment]
We will assume that this was inadvertently malicious.

External contributors judged to be deliberately trying to sabotage our codebase should result in us doing this: https://help.github.com/articles/blocking-a-user-from-your-organization/

Perhaps also with an escalation to our Infosec peeps, because that sort of behaviour might warrant some other sort of response (e.g. police and/or three-letter-agencies)

**Comment by damonjagger** [comment]
Malicious code

**Comment by damonjagger** [comment]
Closed per discussion above.

**Pull request closed without merging**